project1/src/components/FormikInputs/FormikTextField.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import { Field, FieldProps, ErrorMessage } from "formik";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import { withStyles, WithStyles } from "@material-ui/core/styles";
+import { styles } from "./styles/FormikTextFieldStyle";
+import { PropTypes } from "@material-ui/core";
+
+interface FormikTextFieldProps {
+  name: string;
+  label: string;
+  margin: PropTypes.Margin;
+  type?: string;
+}
+
+const FormikTextField: React.FC<
+  FormikTextFieldProps & WithStyles<typeof styles>
+> = props => {
+  return (
+    <Grid item>
```

Comment:
this shouldn't be in the `FormikTextField` component.
As this component is only concerned about wrapping a material ui `TextField` with formik `Field`.
it shouldn't invlove stylings

Reply:
you may wrap `FormikTextField` with another wrapper with `Grid` component